src/main/kotlin/com/terraformation/backend/accelerator/db/ApplicationStore.kt

```diff
@@ -503,7 +503,11 @@ class ApplicationStore(
    *
    * @return The name that was actually used, possibly including a suffix.
    */
-  private fun updateInternalName(applicationId: ApplicationId, internalName: String): String {
+  private fun updateInternalName(
```

Comment:
This function name feels a little misleading now. I wonder if it'd be better to split the country code update out into its own helper function. Then you don't have to pass `null` to this at one of the call sites, which kind of felt like a bit of a code smell to me.